data/migrations/V0108__fix_processed_amendment_chain.sql

```diff
@@ -0,0 +1,3247 @@
+CREATE OR REPLACE VIEW ofec_processed_filing_vw AS
```

Comment:
It may be better to make this view specific for all-but-forms-1-and-2 (is there a better name) and create a similar view for forms-1-and-2 and use the logic from lines 76-95. The materialized view can then be a simple union of these 2 views, making a clear separation between complex computations and collation.

Reply:
@vrajmohan that seems like a good plan. I'll be in training today through Thursday and would be able to make the change by Friday at the latest, but I don't mind if you prefer to get the change in faster.